src/components/Help/index.tsx

```diff
@@ -67,10 +50,12 @@ export default function Help({
 }: PropsWithChildren<HelpProps>) {
   const [id] = useState(() => `help-${nextId++}`)
   const [mounted, setMounted] = useState(false)
+  const [modalRoot, setModalRoot] = useState<Element | null>(null)
```

Comment:
```suggestion
const modalRoot = document.querySelector('#modal-root')
```

You can always get the `#modal-root` without needing to wait for the component to be mounted.

Reply:
I am doing this because generally SSR will complain if it doesn't match, this is why I put this in a useEffect

Reply:
Actually, let me check to see if it complains if i do this.

Reply:
I'll just do it this way to be consistent with other modals, and we can clear it all up if we ever want to tackle react hydration warnings. At least it's not a critical SSR error that would cause a popup during dev.